pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -180,12 +191,39 @@ func (c *K8sOrchestrator) InitTopologyServiceInController(ctx context.Context) (
 					return nil, err
 				}
 
+				isAcceptPreferredDatatsoresFSSEnabled := c.IsFSSEnabled(ctx, common.AcceptPreferredDatastores)
+
 				controllerVolumeTopologyInstance = &controllerVolumeTopology{
-					k8sConfig:                 config,
-					nodeMgr:                   nodeManager,
-					csiNodeTopologyInformer:   *crInformer,
-					clusterFlavor:             clusterFlavor,
-					isCSINodeIdFeatureEnabled: c.IsFSSEnabled(ctx, common.UseCSINodeId),
+					k8sConfig:                             config,
+					nodeMgr:                               nodeManager,
+					csiNodeTopologyInformer:               *crInformer,
+					clusterFlavor:                         clusterFlavor,
+					isCSINodeIdFeatureEnabled:             c.IsFSSEnabled(ctx, common.UseCSINodeId),
+					isAcceptPreferredDatastoresFSSEnabled: isAcceptPreferredDatatsoresFSSEnabled,
```

Comment:
Why not just call IsFSSEnabled here which is aligned with the line above? 

After that, you can use `controllerVolumeTopologyInstance. isAcceptPreferredDatastoresFSSEnabled` to reference it.